google-cloud-video-stitcher/samples/create_cdn_key.rb

```diff
@@ -60,9 +60,14 @@ def create_cdn_key project_id:, location:, cdn_key_id:, hostname:, key_name:, pr
                   }
                 end
 
-  response = client.create_cdn_key parent: parent, cdn_key: new_cdn_key, cdn_key_id: cdn_key_id
+  operation = client.create_cdn_key parent: parent, cdn_key: new_cdn_key, cdn_key_id: cdn_key_id
```

Comment:
I recommend keeping lines shorter than 80 columns, to ensure reasonable formatting when these appear on cloudsite. (Here and a number of additional places below.) Doesn't matter so much for tests above, but for the samples themselves, I think we should try to be be pretty strict about it.

Reply:
Done